string-manipulation-3/src/is-anagram.js

```diff
@@ -0,0 +1,29 @@
+/* exported isAnagram */
+// create a counter variable equal to 0
+// remove spaces from both firstString and secondString and turn them into arrays
+// for each value i from index 0 to final index of firstStringArr
+// for each value j from index 0 to final index of secondStringArr
+// if firstStringArr at current index i is strictly equal to secondStringArr at current index j
+// assign value 0 to secondStringArr at current index j, increment counter, then break
+// if counter is strictly equal to length of firstStringArr and length of firstStringArr is strictly equal to length of secondStringArr, return true
+// else return false
+
+function isAnagram(firstString, secondString) {
+  var counter = 0;
+  var firstStringArr = firstString.split(' ').join('').split('');
+  var secondStringArr = secondString.split(' ').join('').split('');
```

Comment:
This is doing three full loops over each string just to remove spaces. If you make your own for loop to do this, you can do it with a single loop over the string 🤓